src/Language/AST/Node/Contract/DefinitionNodeInterface.php

```diff
@@ -2,7 +2,17 @@
 
 namespace Digia\GraphQL\Language\AST\Node\Contract;
 
+use Digia\GraphQL\Language\AST\Node\NameNode;
+
 interface DefinitionNodeInterface extends NodeInterface
 {
+    /**
```

Comment:
Good improvements. The interfaces are incomplete at this point. Keep up the good work!